tasks/agent.py

```diff
@@ -128,6 +128,7 @@ def build(
     cmake_options='',
     bundle=None,
     bundle_ebpf=False,
+    agent_bin='',
```

Comment:
I'd feel more at ease with `None` as the default value, it should fail in a clearer way in case we end up using it as is.

Reply:
Agree on this point. Both are falsey values, but using `None` makes the intent stand out more.